app/src/main/java/org/mozilla/fenix/searchdialog/SearchDialogFragment.kt

```diff
@@ -389,21 +388,9 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
     ) {
         when (requestCode) {
             REQUEST_CODE_CAMERA_PERMISSIONS -> qrFeature.withFeature {
-                context?.let { context: Context ->
```

Comment:
Do we know why we were safe checking context here? Do we think using `requireContext` instead could be problematic?

Reply:
No, it won't be problematic here